scripts/calculateRevenue/protocols/beefy/index.ts

```diff
@@ -16,7 +20,7 @@ export function getUserTvl(
   txHistory: BeefyInvestorTransactionWithUsdBalance[],
   timestamp: Date,
 ): number {
-  let lastTvl = txHistory[0].usd_balance
+  let lastTvl = 0
```

Comment:
This is a small bugfix. Since we're assuming that the Beefy user TX data is complete, we should be assuming that the user starts out with 0 balance in all vaults.